os.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <stdlib.h>
+#include <time.h>
+
+namespace os {
+	std::string random_sequence(int length) {
+		int i = 0;
+		int randomNumber;
+		char letter;
+		std::string result = "";
```

Comment:
std::string class is empty-string initialized by default;
```suggestion
		std::string result;
```